scene/2d/audio_stream_player_2d.cpp

```diff
@@ -36,6 +36,8 @@
 #include "scene/main/window.h"
 #include "scene/resources/world_2d.h"
 
+#define PARAM_PREFIX "parameters/"
```

Comment:
Can this be defined elsewhere, to avoid repeating it for every player type?
Maybe it could be `constexpr String` (or `char *` if String can't be used) in AudioStream?

Reply:
I think we have to just fuse all 3 in a common internal class at this rate, I am not sure where I would place it.

Reply:
Maybe a .inc file for common classes?

Reply:
#87061